examples/ext/cooldowns/custom_bucket.py

```diff
@@ -0,0 +1,25 @@
+from typing import Optional, Tuple
+
+import hikari
+
+import crescent
+from crescent.ext import cooldowns
+
+bot = crescent.Bot("...")
+
+
+def guild_specific_rate_limits(
```

Comment:
Since this is all developer facing I think we could go with `member_bucket` instead. Up to you though

Reply:
I'm pretty sure members have the same ID as Users so member bucket wouldn't be as clear to what the function does

Reply:
I still think "guild_specific" is more clear